projects/pastanaga-angular/src/lib/toast/toast.service.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { createComponentFactory, Spectator } from '@ngneat/spectator/jest';
+import { ToastService } from './toast.service';
+import { BehaviorSubject } from 'rxjs';
+import { ToastStatus } from './toast.model';
+import { ApplicationRef, Component, ComponentRef, Renderer2 } from '@angular/core';
+import { ToastComponent } from './toast.component';
+import { BrowserDynamicTestingModule } from '@angular/platform-browser-dynamic/testing';
+import { PaToastModule } from '../..';
+
+@Component({
+    template: ``,
+})
+export class TestComponent {
+    constructor(private service: ToastService) {}
+}
+
+describe('ToastService', () => {
+    let nextId = 0;
+    const createService = createComponentFactory({
```

Comment:
you don't need to create a component to test this service, you should use spectator service factory and just check the method which create the DOM are called.

Reply:
fact is we need to use overrideModules in order to define BrowserDynamicTestingModule, which is only available in createComponentFactory for now